src/System.Private.ServiceModel/tools/test/Bridge.Build.Tasks.targets

```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <UsingTask AssemblyFile="$(WcfToolsOutputPath)Bridge.Build.Tasks.dll" TaskName="ReleaseBridgeResourcesTask" />
+
+  <!-- ReleaseBridgeResources:
+      This target asks the Bridge (if running) to release its resources.
+      This is necessary so that the content of the resource folder can change.
+  -->
+  <Target Name="ReleaseBridgeResources">
+    <ReleaseBridgeResourcesTask
+      BridgeHost="$(BridgeHost)"
```

Comment:
What about the scenario where the Bridge is running on another host? With a shared bridge, a normal build on one machine could release resources currently being used by tests running on another host.